canonical/jaas-dashboard
# WD-672 - Move polling to middleware

## Done

- Move the model/controller polling to the middleware.
- Add polling tests.

## QA

- Pull code
- Run `dotrun clean && dotrun serve`
- Open http://0.0.0.0:8036/
- Check that you can log in and connect to models.
